public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+  <body>
+    <div id="app">
+      Humble Bundle
+      <div id="charity">
+        <script src="http://localhost:3987/static/js/bundle.js"></script>
+      </div>
+      <div id="contributors">
+        Contributors
+      </div>
+    </div>
+  </body>
+</html>
```

Comment:
The red circle with the minus sign is asking for a new line at the end of the file. Same comment re 
src/app.jsx and webpack.config.js.